search_username/catalog/models.py

```diff
@@ -0,0 +1,13 @@
+from sqlalchemy import Column, Integer, String
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base()
+
+
+class SaveUsers(Base):
```

Comment:
Bad name for the model.
Please read recommendations for models naming.

Reply:
renamed